scripts/cmake/sdk_windows.cmake

```diff
@@ -0,0 +1,840 @@
+defold_log("sdk_windows.cmake:")
```

Comment:
This one is very large, if feels excessive to be honest, and should probably be rewritten to make it more readable.